### PR TITLE
17 move ui from task list to command

### LINKED ISSUE #17: Move UI stuff from TaskList to Command

### LINKED ISSUE #24: Parser and Command: Fix checkstyle issues

### PR DESCRIPTION
**Pull request by amanda-chua**

### CODE REVIEW

File: src/main/java/athena/ListCommand.java [hunk 1]
```diff
@@ -10,12 +10,14 @@ public ListCommand(String importance) {
         taskImportance = importance;
     }
     /**
-     * Calls Ui to print the list of tasks.
+     * Calls TaskList to filter the list based on importance and
+     * calls Ui to print the list of tasks.
```
Comment (daniellimws) [inline src/main/java/athena/ListCommand.java]: Need an empty line before method definition.

```
Error: eckstyle] [ERROR] /home/runner/work/tp/tp/src/main/java/athena/ListCommand.java:17: 'METHOD_DEF' should be separated from previous statement. [EmptyLineSeparator]
```

File: src/main/java/athena/AddCommand.java [hunk 2]
```diff
@@ -23,13 +24,17 @@ public AddCommand(String name, String startTime, String duration, String deadlin
...
+     * calls Ui to print out the task added.
      * @param taskList Tasks List
      * @param ui Ui
      */
     @Override
     public void execute(TaskList taskList, Ui ui) {
-        taskList.addTask(taskName, taskStartTime,taskDuration, taskDeadline, taskRecurrence, taskImportance, taskNotes);
+        taskList.addTask(taskName, taskStartTime,taskDuration, taskDeadline,
+                taskRecurrence, taskImportance, taskNotes);
+        ui.printTaskAdded(taskName, taskStartTime,taskDuration, taskDeadline,
```
Comment (daniellimws) [inline src/main/java/athena/AddCommand.java]: Need a space before `taskDuration`.

File: src/main/java/athena/AddCommand.java [hunk 3]
```diff
@@ -23,13 +24,17 @@ public AddCommand(String name, String startTime, String duration, String deadlin
...
-     * Adds a task to the Tasks list.
+     * Adds a task to the Tasks list and
+     * calls Ui to print out the task added.
      * @param taskList Tasks List
      * @param ui Ui
      */
     @Override
     public void execute(TaskList taskList, Ui ui) {
-        taskList.addTask(taskName, taskStartTime,taskDuration, taskDeadline, taskRecurrence, taskImportance, taskNotes);
+        taskList.addTask(taskName, taskStartTime,taskDuration, taskDeadline,
```
Comment (daniellimws) [inline src/main/java/athena/AddCommand.java]: Need a space before `taskDuration`.

File: src/main/java/athena/EditCommand.java [hunk 4]
```diff
@@ -22,12 +23,16 @@ public EditCommand(int index, String name, String startTime, String duration, St
...
-     * Edits a task from the Tasks list.
+     * Edits a task from the Tasks list and
+     * calls Ui to print task edited.
      * @param taskList Tasks List
      * @param ui Ui
      */
     @Override
     public void execute(TaskList taskList, Ui ui) {
-         taskList.editTask(taskIndex, taskName, taskStartTime,taskDuration, taskDeadline, taskRecurrence, taskImportance, taskNotes);
+        taskList.editTask(taskIndex, taskName, taskStartTime,taskDuration, taskDeadline,
```
Comment (daniellimws) [inline src/main/java/athena/EditCommand.java]: Need a space before `taskDuration`.

File: src/main/java/athena/EditCommand.java [hunk 5]
```diff
@@ -22,12 +23,16 @@ public EditCommand(int index, String name, String startTime, String duration, St
...
+                taskRecurrence, taskImportance, taskNotes);
+        ui.printTaskEdited(taskIndex, taskName, taskStartTime,taskDuration, taskDeadline,
```
Comment (daniellimws) [inline src/main/java/athena/EditCommand.java]: Need a space before `taskDuration`.

**Review by daniellimws — CHANGES_REQUESTED** [review]
Looks good. Left some comments about code quality.

I think we wait until #29 is merged before merging this so that the build doesn't break.

**Review by wish2023 — APPROVED** [review]

File: src/main/java/athena/Parser.java [hunk 6]
```diff
@@ -1,5 +1,7 @@
 package athena;
 
+import athena.commands.*;
```
Comment (daniellimws) [inline src/main/java/athena/Parser.java]: Prof said cannot using `*` to import. Refer to https://github.com/nus-cs2113-AY2021S1/forum/issues/36

**Review by daniellimws — APPROVED** [review]
Looks nice. Once you change the `import` statement I think it is good to merge.

**Pull request merged**